config/karma/test.karma.conf.js

```diff
@@ -11,8 +11,11 @@ const minimist = require('minimist');
 function getConfig(config) {
   require('./shared.karma.conf')(config);
 
-  const argv = minimist(process.argv.slice(2));
+  const argv = minimist(process.argv.slice(2), {
+    boolean: ['browser', 'enableDesktopNotifications']
+  });
   const browser = (argv.headless) ? 'ChromeHeadless' : 'Chrome';
+  const enableDesktopNotifications = (argv.enableDesktopNotifications);
```

Comment:
What purpose do the parenthesis serve?  I'd also be ok with just checking for `argv.enableDesktopNotifications` without creating a variable.

Reply:
None, probably. I was following the pattern of the line above ... That whole line is removed in my second commit.